core/src/main/scala/opal/PurityAnalysis.scala

```diff
@@ -70,7 +73,9 @@ object PurityAnalysis extends DefaultOneStepAnalysis {
       classFile <- project.allProjectClassFiles.par
       method <- classFile.methods
     } {
-      pool.execute(() => analyze(project, methodToCellCompleter, classFile, method))
+      pool.triggerExecution(methodToCell(method)) //is not visible, but we can create a helper cell that triggers
+      //      pool.createCell[PurityKey.type, Purity](PurityKey, () => NoOutcome)
+      //        .whenNext(methodToCell(method), _ => NoOutcome)
```

Comment:
Not sure what you mean. In any case, I don't think the test case should call `triggerExecution`, since it's not public.

Reply:
Sorry, this was confusing. I removed this comment.